src/cli/requirements.txt

```diff
@@ -1,17 +1,19 @@
-msal~=1.10.0
+msal~=1.12.0
 requests~=2.25.1
 jmespath~=0.10.0
 semver~=2.13.0
 signalrcore==0.9.2
-asciimatics~=1.12.0
+asciimatics~=1.13.0
 pydantic~=1.8.1 --no-binary=pydantic
 memoization~=0.3.1
 msrestazure==0.6.4
 azure-storage-blob~=12.8
 azure-applicationinsights==0.1.0
-tenacity==6.3.1
-docstring_parser==0.7.3
-azure-cli-core==2.26.0
+tenacity==8.0.1
+docstring_parser==0.8.1
+azure-cli-core==2.26.1
+# packaging is required but not specified by azure-cli-core
```

Comment:
Is there an `azure-cli-core` issue we can link to?

Reply:
Not at t his time.

I am working on a PR for azure-cli to include it, but probably won't be done until later this week.